pokeapp/src/utilities/ApiCall.js

```diff
@@ -1,12 +1,17 @@
 const Pokedex = require("pokeapi-js-wrapper");
-const P = new Pokedex.Pokedex();
 
 const interval = {
   offset: 0,
   limit: 1119,
-  cache: true,
 };
 
+const customOptions = {
+  cache: true,
+}
+
+
+const P = new Pokedex.Pokedex(customOptions);
```

Comment:
It's just me, but maybe give a better descriptive variable? But, if it works then it works 😘

Reply:
Thanks for the input!